addons/l10n_mx/data/template/account.tax-mx.csv

```diff
@@ -51,10 +51,10 @@
 "","","","","","","","","","","","","","","","tax","invoice","cuenta216_10_20","-DIOT: Retención","",""
 "","","","","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","","","","tax","refund","cuenta216_10_20","+DIOT: Retención","",""
-"tax18","13","1.25% WH","VAT witholding RESICO 1.25%","Retención IVA(-1.25%)","-1.25","percent","purchase","tax_group_iva_ret_1_25","on_payment","cuenta216_10","","iva","Tasa","0","base","invoice","","","Retención de IVA RESICO 1.25%","Retención IVA(-1.25%)"
-"","","","","","","","","","","","","","","","tax","invoice","cuenta216_10_20","-DIOT: Retención","",""
+"tax4","13","1.25% WH","Witholding of income tax RESICO 1.25%","Retención ISR(-1.25%)","-1.25","percent","purchase","tax_group_iva_isr_ret_1_25","on_invoice","cuenta216_04","","isr","Tasa","0","base","invoice","","","Retención de ISR RESICO 1.25%","Retención ISR(-1.25%)"

```

Comment:
```suggestion
"mx_wh_1_25","13","1.25% WH","Witholding of income tax RESICO 1.25%","Retención ISR(-1.25%)","-1.25","percent","purchase","tax_group_iva_isr_ret_1_25","on_invoice","cuenta216_04","","isr","Tasa","0","base","invoice","","","Retención de ISR RESICO 1.25%","Retención ISR(-1.25%)"

```
I'd use another id (even if it's not very consistent) for safety, this xmlid looks like something we probably removed at some point already, which could cause issues.

Reply:
That makes sense, updated the PR :)